### PR TITLE
Refactor Review Controller

### LINKED ISSUE #18: Check Review MVC for data control

### PR DESCRIPTION
**Pull request by helloeduardo**
Resolves #18 

Refactors controller, encapsulating extraneous functionality into other methods.
All tests passing

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -1,44 +1,35 @@
 class ReviewsController < ApplicationController
-  before_action :set_review, only: [:edit, :update, :destroy]
+  before_action :find_review, only: [:edit, :update, :destroy]
```
Comment (ETBassist) [inline app/controllers/reviews_controller.rb:2]: Ooooh very nice, will have to see where else we can use this

**Review by ETBassist — APPROVED** [review]
Wayne's happy

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -1,44 +1,35 @@
...
   end
 
   def create
-    user = User.find_by(name: params[:user_name])
-    shelter = Shelter.find(params[:shelter])
-    if Review.check_exists(user, shelter)
-      review = Review.check_exists(user, shelter)
-      flash.notice = 'You already have a review, want to update it?'
-      redirect_to "/reviews/#{review.id}/edit"
+    return redirect_to_existing_review if Review.check_exists(@user, @shelter)
```
Comment (katemorris) [inline app/controllers/reviews_controller.rb:10]: You.are.a.genius

**Review by katemorris — APPROVED** [review]
You are my knight in shining ... ruby? :)

**Pull request merged**